libs/styles/styles.css

```diff
@@ -127,6 +128,24 @@
   --icon-size-xs: 24px;
   --icon-size-xxs: 16px;
 
+  
```

Comment:
@mokimo - These values are going to be shared. (aside, hero, editorial) I actually took this out of the work @meganthecoder is doing w/ Asides so that we can share the same styles. I suggested that we load these in a separate file as well and we both thought the extra fetch request was more taxing then the extra styles. I think it's debatable, but on second thought, I think loading this as an extra conditional style sheet it likely more scalable so we only load when needed. I will refactor this and coordinate w/ the other related eng. efforts. Thanks

Reply:
I moved this into a `iconography.css` file and conditionally loaded it. Open to suggestions on name.  
Note on name: There are some additional styles that will be added to this to support .avatar style/sizes so It will include (lockup, avatar) etc. @meganthecoder